site/en/docs/extensions/whatsnew/index.md

```diff
@@ -30,18 +16,79 @@ date: 2021-02-25
 Check this page often to learn about changes to the Chrome extensions platform,
 its documentation, and related policy or other changes.
 
-### 2021.06.08: Introducing chrome.scripting (blog post)
+### New "Introducing chrome.scripting" blog post
+#### Published 2021.06.08
 
 The [Scripting API](/docs/extensions/reference/scripting/) is a new Manifest V3 API focused on,
 well, scripting. In this post we dig into the motivations for this change and take a closer look at
 some of the new capabilities it introduces. [Read the post](/blog/crx-scripting-api).
 
-### 2021.01.19: Manifest V3 launched
+### ES modules for service workers
+#### Launched in Chrome 91
+
+JavaScript supports modules in service workers. In your manifest, specify a module in your manifest:
```

Comment:
Nit

```suggestion
Chrome now supports modules in service workers. In your manifest, specify a module in your manifest:
```

Terminology is a slightly wonky here. The JavaScript language doesn't have a concept of service workers or even workers. "Web Workers" (which service workers inherit from) were originally specified in the W3C and are currently maintained as part of the [HTML Living Standard](https://html.spec.whatwg.org/#toc-workers).